JdbcGtfsExporter: cleanUpZipFile() fails on Windows

On Windows, `JdbcGtfsExporter.cleanUpZipFile()` fails with an `IllegalArgumentException` as the method it uses for generating a URI works only on systems with UNIX-like file paths.

With [Data Tools](https://github.com/conveyal/datatools-server) deployed on Windows, this means exporting a GTFS feed partially fails with a log message like the following:

```
Dec 30 18:05:41.308 [pool-1-thread-2] ERROR c.c.gtfs.loader.JdbcGtfsExporter:310 - Exception while creating snapshot: java.lang.IllegalArgumentException: Illegal character in opaque part at index 13: jar:file://C:\Users\simon\AppData\Local\Temp\snapshot3620290097384378083zip
java.lang.IllegalArgumentException: Illegal character in opaque part at index 13: jar:file://C:\Users\simon\AppData\Local\Temp\snapshot3620290097384378083zip
        at java.net.URI.create(URI.java:852)
        at com.conveyal.gtfs.loader.JdbcGtfsExporter.cleanUpZipFile(JdbcGtfsExporter.java:328)
        at com.conveyal.gtfs.loader.JdbcGtfsExporter.exportTables(JdbcGtfsExporter.java:301)
        at com.conveyal.datatools.editor.jobs.ExportSnapshotToGTFSJob.jobLogic(ExportSnapshotToGTFSJob.java:51)
        at com.conveyal.datatools.common.status.MonitorableJob.run(MonitorableJob.java:126)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
Caused by: java.net.URISyntaxException: Illegal character in opaque part at index 13: jar:file://C:\Users\simon\AppData\Local\Temp\snapshot3620290097384378083zip
        at java.net.URI$Parser.fail(URI.java:2848)
        at java.net.URI$Parser.checkChars(URI.java:3021)
        at java.net.URI$Parser.parse(URI.java:3058)
        at java.net.URI.<init>(URI.java:588)
        at java.net.URI.create(URI.java:850)
        ... 7 more
Dec 30 18:05:41.326 [pool-1-thread-2] ERROR c.c.d.e.jobs.ExportSnapshotToGTFSJob:54 - Error (Illegal character in opaque part at index 13: jar:file://C:\Users\simon\AppData\Local\Temp\snapshot3620290097384378083zip) encountered while exporting database tables.
```

Although the exported GTFS feed is written, the clean-up step meant to remove empty files from its contents is clearly skipped.

The culprit is this call at [line 328 of `JdbcGtfsExporter.java`](https://github.com/conveyal/gtfs-lib/blob/cab706fbe03c1f1b1696d7090f1961268e5b5b4d/src/main/java/com/conveyal/gtfs/loader/JdbcGtfsExporter.java#L328), which attempts to build a URI directly from a file path:

```
URI zip_disk = URI.create("jar:file://" + outFile);
```
This happens to produce a valid URI on UNIX-like systems, but fails on Windows as it requires [a unique convention for file URIs](https://en.wikipedia.org/wiki/File_URI_scheme#Windows_2).

--- a/src/main/java/com/conveyal/gtfs/loader/JdbcGtfsExporter.java
+++ b/src/main/java/com/conveyal/gtfs/loader/JdbcGtfsExporter.java
@@ -11,6 +11,7 @@ import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import javax.sql.DataSource;
+import java.io.File;
 import java.io.FileOutputStream;
 import java.io.FilterOutputStream;
 import java.io.IOException;
@@ -325,7 +326,8 @@ public class JdbcGtfsExporter {
         zip_properties.put("create", "false");
 
         // Specify the path to the ZIP File that you want to read as a File System
-        URI zip_disk = URI.create("jar:file://" + outFile);
+        // (File#toURI allows this to work across different operating systems, including Windows)
+        URI zip_disk = URI.create("jar:" + new File(outFile).toURI());
 
         // Create ZIP file System
         try (FileSystem fileSystem = FileSystems.newFileSystem(zip_disk, zip_properties)) {

